src/tween/tweenable.babel.js

```diff
@@ -10,35 +10,39 @@ import Module   from '../module';
   @class Tweenable
 */
 class Tweenable extends Module {
+
   /*
     `play` method for the timeline.
     @public
     @param {Number} Time shift.
```

Comment:
Maybe could re-add the shift argument in play, playBackward, resume, setProgress and setSpeed? Or is it unnecessary when using the argument object?

We are currently passing in the arguments to the timepline,play.apply functions. And the play function (defined in tween.babel.js), is taking a shift argument. So I guess we could add all these shift back to add back support for the shift parameter. But think it's a bit unnecessary to use the arguments object when it's just a single argument 🤔. And also a bit hard to read.

This comment is also related to all other functions in this file that has the same argument.
@xavierfoucrier

Reply:
This review target a blank line, you mean parameters  to the public "play" method? :thinking:

Reply:
Sorry, was the line right before, the `@param {Number} Time shift.`

Reply:
If you provide a time shift argument, ex myTimeline.play(2), will it still work course we pass the arguments object to the timeline function, even though the argument is not explicitly defined in the play() declaration?